packages/protocol-contracts/contracts/testing/TestContractsUniswapV3.sol

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Can't we import this from `v3-periphery/NonfungiblePositionManager.sol`?

Reply:
I will create a fake contract to import it, because we just want the types